bashrc.formbuilder

```diff
@@ -9,7 +9,7 @@
 # Use the globally installed node and its modules
 LFORMS_NODE=node-v14
 # Set path
-PATH=~/${LFORMS_NODE}/bin:/bin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/etc
+PATH=~/${LFORMS_NODE}/bin:${PATH}
```

Comment:
This could lead to unwanted path settings, especially when working with multiple projects in a terminal window. The idea is to keep the PATH clean and consistent irrespective of how many times the file is sourced.

We know it largely depends on user's work flow. We suggest they make local changes as they need.

Reply:
I see, okay, I will revert this change.